zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/config/ExporterConfiguration.java

```diff
@@ -226,4 +237,100 @@ public String toString() {
       return "BulkConfiguration{" + "delay=" + delay + ", size=" + size + '}';
     }
   }
+
+  public static class ArchiverConfiguration {
+    private boolean rolloverEnabled = true;
+    private String rolloverDateFormat = "yyyy-MM-dd";
+    private String elsRolloverDateFormat = "date";
+    private String rolloverInterval;
```

Comment:
:question: Why drop the default value here? I can see in the original properties it's "1d"?

Reply:
I had accidentally copied the "no value" from task list instead of operator I think.